tests/ext/ddtrace_clocks.phpt

```diff
@@ -0,0 +1,12 @@
+--TEST--
+Test that clocks return > 0
+--FILE--
+<?php
+var_dump(ddtrace_monotonic_nsec() > 0);
```

Comment:
I'm not sure how I feel about exposing these to userland unless we are using them in `DDTrace\Time`. Would you be OK with holding off exposing these userland functions until we need to use them in userland?

Reply:
Very reasonable; will get these removed tomorrow morning for another CR. Thanks, Sammy!